data/test.mod

```diff
@@ -10,8 +10,8 @@ NEURON  {
 }
 
 STATE {
```

Comment:
The test now checks only the case where units are provided.

A more low level unit test where we test correct parsing of state variable definitions with and without units is a good idea.